molecule/default/verify.yml

```diff
@@ -10,6 +10,7 @@
   tasks:
     - set_fact:
         vrfy_maillog_path: "{{ vrfy_maillog_path_families[ansible_os_family] }}"
+        vrfy_syslog: /var/log/syslog
```

Comment:
@Chithrak07 this needs to be conditional the same way `vrfy_maillog_path` is. Syslog writes to `/var/log/messages` on red hat.